stock_picking_info_lot/models/stock_move_line.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    lot_info = fields.Char("Lot/Serial Number info")
+    lot_info_usage = fields.Selection(
+        [("no", "No"), ("optional", "Optional"), ("required", "Required")],
+        related="product_id.product_tmpl_id.lot_info_usage",
+    )
```

Comment:
Fix the log warning `odoo.fields: stock.move.line.lot_info_usage: selection attribute will be ignored as the field is related`
```suggestion
    lot_info_usage = fields.Selection(related="product_id.product_tmpl_id.lot_info_usage")
```